lib/item.js

```diff
@@ -280,6 +280,8 @@ Item.load = function*(args) {
 	var data = yield models.items
 		.find({ where: criteria })
 		.complete(gx.resume);
+	
+	if (!data) return;
```

Comment:
This is how we handle a `Item` which cannot be created from the `data`. Seems appropriate to do the same here.

Reply:
See https://github.com/dchester/showcase/pull/34/files#diff-d742b5c2be96bad795c004074f588738R288

Reply:
Deepak suggested we caught errors at the highest level and handle them gracefully rather than protecting against this single failure mode.